frontend/src/metabase/visualizations/lib/settings/graph.js

```diff
@@ -467,7 +467,9 @@ export const GRAPH_AXIS_SETTINGS = {
   },
   "graph.x_axis.scale": {
     section: t`Axes`,
-    title: t`X-axis scale`,
+    group: t`X-AXIS`,
```

Comment:
[There's a thread on Slack](https://metaboat.slack.com/archives/C03SD8HDJLC/p1665569319645579) going on about this.

Basically:
1. I want to confirm if all caps are the new styles for group labels in visualization settings.
2. That we should create this string in the Title case, but make the text look uppercase from CSS. As seen in the side navbar e.g. `BOOKMARKS`, and `COLLECTIONS`
![image](https://user-images.githubusercontent.com/1937582/195317575-b68a70f1-dcb0-463b-9eb7-8beae7d0ba68.png)
![image](https://user-images.githubusercontent.com/1937582/195317738-0f63782b-9a98-4d34-9bfc-70c59b359375.png)

I also saw you have set the CSS to make this uppercase.
https://github.com/metabase/metabase/pull/25886/files#diff-e0192eb22397f18e48d5e45c71ba38d38a8e1dbb9f50640ddd4f72a731d693d1R11